net/potd/Makefile

```diff
@@ -0,0 +1,39 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=potd
+PKG_VERSION:=1.0
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/lnslbrty/potd.git
+PKG_SOURCE_VERSION:=146cc80a3333297a614966a3a001ca96fa719ad9
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION)-$(PKG_SOURCE_VERSION).tar.gz
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_MIRROR_HASH:=8da25c5ba87548d1b97fb112516f3c7c01b6347583fbaa0ffbf8b92288c73cf0
+PKG_MAINTAINER:=Toni Uhlig <matzeton@googlemail.com>
+
+PKG_INSTALL:=1
+PKG_FIXUP:=autoreconf
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/potd
+	SECTION:=net
+	CATEGORY:=Network
+	DEPENDS:=+libseccomp +libssh +@KERNEL_NAMESPACES +@KERNEL_UTS_NS +@KERNEL_IPC_NS \
+	         +@KERNEL_PID_NS +@KERNEL_NET_NS +@KERNEL_CGROUPS +@KERNEL_CGROUP_PIDS
```

Comment:
Please don't enable kernel config options by default, either only depend on them (`@KERNEL_...` without the `+`) or add a `DEFAULT:=n` so the package isn't built and selected by default.